### ISSUE TITLE
Refine edge and content alignment

### ISSUE THREAD

**Issue by iandunn** [WordPress/wporg-news-2021#37]
There are a couple related discussions in closed threads, so consolidating here.

* Edge alignment inconsistent - https://github.com/WordPress/wporg-news-2021/pull/28#discussion_r717030878
* Content column no longer aligned w/ post title - https://github.com/WordPress/wporg-news-2021/commit/fa25d31b2dbaf3bde87a3d90cb568e0505cd4c98#r57269081

### goals

- [ ] all top-level elements should align on their left/right edges, including local header. only exception is global header.
    * text in post content isn't justified, so the right edge won't appear to align, but the container does
    * body text extra padding to push towards center of viewport. characters per line on desktop can be as high as `121`
- [ ] global header elements align to edges of screen, with a small gap
- [ ] `alignwide`
    * quotes - same edge padding as top-level elements, text inside has larger left margin (not aligned w/ anything). characters per line on desktop about `48`
        * on mobile, the background image is offset so that half of the first mark of off hidden
- [ ] `alignfull`:
    * images and videos - go to viewport edge
    * normal text - background goes to viewport edge, text is aligned w/ other page elements
         * characters per line on desktop is as high as `116`
    * verse - background fills content area, text inside has similar padding to edge padding
- [ ] in general, mobile is same as above, just with smaller edge padding 

@beafialho, @coreymckrill, does all that seem right to you?

@beafialho, what are your thoughts on the characters per line? they seem much higher than the usual guideline

### implementation

- [ ] It seems to make the most sense to remove all left/right padding/margin from all top level components, and then set a single rule -- ie, `body > * {...}` -- to apply the new one. the header can overwrite it, or we could do `*:not(.site-header)` or whatever.
- [ ] also move left/right padding from `alignfull` to  only _text_ children (not images/videos/etc)
- [ ] and then make all the various changes to implement the details of the mockup.

any thoughts @coreymckrill ?

**Comment by coreymckrill** [comment]
This is a little hard for me to visualize, but it _sounds_ like what I see in the design mockups.

**Comment by beafialho** [comment]
> @beafialho, what are your thoughts on the characters per line? they seem much higher than the usual guideline

Yes, indeed @iandunn @coreymckrill, I should have asked this before because it is not ideal as screen size increases. 

Is it possible to make the body text get bigger as the screen size gets bigger?

<img width="1417" alt="Captura de ecrã 2021-10-01, às 15 45 02" src="https://user-images.githubusercontent.com/65220155/135641097-43c4ab73-dd5e-4626-a8b2-394128dc4343.png">

For example, while at 1320px screen size we have a post in a Regular Paragraph `16pt` , at 2000px can it turn into an Extra Large Paragraph `24pt` as a way to facilitate reading? Would it be helpful if I designed what these changes would look like in different breakpoints like I did with the Global Header?
